### PR TITLE
fix: allow adding comments from forks

### PR DESCRIPTION
**Pull request by fzipi**
## what

- adding comments from forks will fail unless we use the `pull_request_target` event

## why

- quantitative run needs to add comments to the PR

### CODE REVIEW

**Review by M4tteoP — APPROVED** [review]
Thank you!

File: .github/workflows/quantitative.yaml
Comment (theseion) [inline .github/workflows/quantitative.yaml:1]: You should also add explicit permissions, i.e. `permissions: none` globally, and
```
permissions:
  pull-requests: write
```
for the job.

**Review by theseion — CHANGES_REQUESTED** [review]

File: .github/workflows/quantitative.yaml [hunk 1]
```diff
@@ -19,6 +19,22 @@ on:
 env:
   GO_FTW_VERSION: '1.1.1'
 
+permissions:
```
Comment (theseion) [inline .github/workflows/quantitative.yaml:22]: This can be replaced with `permissions: {}` (sorry, had it wrong in my head).

**Review by theseion — CHANGES_REQUESTED** [review]

**Review by theseion — APPROVED** [review]

File: .github/workflows/quantitative.yaml [hunk 2]
```diff
@@ -13,6 +13,8 @@ on:
 env:
   GO_FTW_VERSION: '1.1.1'
 
+permissions: {}
+
```
Comment (fzipi) [inline .github/workflows/quantitative.yaml:17]:
```suggestion
```

**Review by M4tteoP — APPROVED** [review]

**Pull request merged**